minecraft/java/spigot/egg-spigot.json

```diff
@@ -22,9 +26,9 @@
     },
     "scripts": {
         "installation": {
-            "script": "#!\/bin\/ash\r\n# Spigot Installation Script\r\n#\r\n# Server Files: \/mnt\/server\r\n\r\napk add --no-cache --update curl git openjdk8 openssl bash nss\r\n\r\n## Only download if a path is provided, otherwise continue.\r\nif [ ! -z \"${DL_PATH}\" ]; then\r\n    cd \/mnt\/server\r\n\r\n    MODIFIED_DOWNLOAD=`eval echo $(echo ${DL_PATH} | sed -e 's\/{{\/${\/g' -e 's\/}}\/}\/g')`\r\n    curl -sSL -o ${SERVER_JARFILE} ${MODIFIED_DOWNLOAD}\r\nelse\r\n    cd \/srv\/\r\n    \r\n    wget https:\/\/hub.spigotmc.org\/jenkins\/job\/BuildTools\/lastSuccessfulBuild\/artifact\/target\/BuildTools.jar\r\n    \r\n    mv BuildTools.jar \/srv\/\r\n\r\n    java -jar BuildTools.jar --rev ${DL_VERSION}\r\n\r\n    mv spigot-*.jar \/mnt\/server\/${SERVER_JARFILE}\r\nfi",
-            "container": "alpine:3.9",
-            "entrypoint": "ash"
+            "script": "#!\/bin\/bash\r\n# Spigot Installation Script\r\n#\r\n# Server Files: \/mnt\/server\r\n\r\napt update \r\napt install -y curl wget git openssl bash bc\r\n\r\nif [ \"${DL_VERSION}\" == \"latest\" ] || (( $(echo \"${DL_VERSION} >= 1.17\" | bc -l) )); then\r\n    mkdir -p \/usr\/local\/java\/16\r\n    curl -Lo java16.tar.gz https:\/\/github.com\/AdoptOpenJDK\/openjdk16-binaries\/releases\/download\/jdk-16.0.1%2B9_openj9-0.26.0\/OpenJDK16U-jdk_x64_linux_openj9_16.0.1_9_openj9-0.26.0.tar.gz\r\n    tar -xzvf java16.tar.gz -C \/usr\/local\/java\/16 --strip-components=1\r\n    export PATH=\/usr\/local\/java\/16\/bin:$PATH\r\n    export JAVA_HOME=\/usr\/local\/java\/16\r\nelse\r\n    mkdir -p \/usr\/local\/java\/8\r\n    curl -Lo java8.tar.gz https:\/\/github.com\/AdoptOpenJDK\/openjdk8-binaries\/releases\/download\/jdk8u282-b08_openj9-0.24.0\/OpenJDK8U-jre_x64_linux_openj9_8u282b08_openj9-0.24.0.tar.gz\r\n    tar -xzvf java8.tar.gz -C \/usr\/local\/java\/8 --strip-components=1\r\n    export PATH=\/usr\/local\/java\/8\/bin:$PATH\r\n    export JAVA_HOME=\/usr\/local\/java\/8\r\nfi\r\n\r\n## Only download if a path is provided, otherwise continue.\r\nif [ ! -z \"${DL_PATH}\" ]; then\r\n    cd \/mnt\/server\r\n\r\n    MODIFIED_DOWNLOAD=`eval echo $(echo ${DL_PATH} | sed -e 's\/{{\/${\/g' -e 's\/}}\/}\/g')`\r\n    curl -sSL -o ${SERVER_JARFILE} ${MODIFIED_DOWNLOAD}\r\nelse\r\n    mkdir -p \/srv\/\r\n    cd \/srv\/\r\n    \r\n    wget https:\/\/hub.spigotmc.org\/jenkins\/job\/BuildTools\/lastSuccessfulBuild\/artifact\/target\/BuildTools.jar\r\n    \r\n    java -jar BuildTools.jar --rev ${DL_VERSION}\r\n\r\n    mv spigot-*.jar \/mnt\/server\/${SERVER_JARFILE}\r\nfi",
+            "container": "debian:buster-slim",
```

Comment:
just use openjdk:16-jdk-alpine and remove from script the apk install javajdk8

Reply:
Java 8 is required for older spigot versions.

Reply:
Okay, I tested it
Buildtools need java 8 to build the older versions

Reply:
So, shut we add a new agg for newer spigot versions?
Or we need the check with version we are using and downloading the correct java version.